drivers/soundwire/bus.c

```diff
@@ -434,6 +443,41 @@ sdw_bwrite_no_pm(struct sdw_bus *bus, u16 dev_num, u32 addr, u8 value)
 	return sdw_transfer(bus, &msg);
 }
 
+int
+sdw_bread_no_pm_unlocked(struct sdw_bus *bus, u16 dev_num, u32 addr)
```

Comment:
Why "int" on a separate line? Does not seem to match style in rest of the file, nor in kernel in general.

Reply:
fixed